.github/workflows/update-matrix-room.yml

```
@@ -18,9 +18,9 @@ jobs:
           URL_TOPIC: 'https://matrix-client.matrix.org/_matrix/client/r0/rooms/!HBsLKoOIEEOkxAeQvC%3Amatrix.org/state/m.room.topic/'
         run: |
           TAGS="$(curl https://api.github.com/repos/opencast/opencast/tags | jq -r '.[].name')"
-          MAJOR="$(echo "$TAGS" | sed 's/\..*$//' | sort -h | tail -n1)"
-          STABLE="$(echo "$TAGS" | sort -h | tail -n1)"
-          LEGACY="$(echo "$TAGS" | grep -v "^${MAJOR}" | sort -h | tail -n1)"
+          MAJOR="$(echo "$TAGS" | sed 's/\..*$//' | sort -version-sort | tail -n1)"
+          STABLE="$(echo "$TAGS" | sort -version-sort | tail -n1)"
```

Didn't test this, but this should be:
```suggestion
          MAJOR="$(echo "$TAGS" | sed 's/\..*$//' | sort --version-sort | tail -n1)"
          STABLE="$(echo "$TAGS" | sort --version-sort | tail -n1)"
```